.kokoro/presubmit/presubmit.cfg

```diff
@@ -5,3 +5,7 @@ env_vars: {
     key: "RUN_SYSTEM_TESTS"
     value: "false"
 }
+env_vars: {
+    key: "RUN_SNIPPETS"
```

Comment:
I think we need to allow this envvar in the Kokoro config in google3.
Also how about `RUN_SNIPPETS_TESTS` for consistensy?

Reply:
`RUN_SNIPPETS_TESTS` sounds good to me